Mathlib/AlgebraicTopology/DoldKan/Equivalence.lean

```diff
@@ -0,0 +1,183 @@
+isomorphism using the fact that `N₂` reflects isomorphisms, and because we can show
+that the composition `N₂ ⟶ N₂ ⋙ Γ₂ ⋙ N₂ ⟶ N₂` is the identity (see `identity_N₂`). The fact
+that `N₂` is defined as a formal direct factor makes the proof easier because we only
+have to compare endomorphisms of an alternating face map complex `K[X]` and we do not
+have to worry with inclusions of kernel subobjects.
+
+In `EquivalenceAdditive.lean`, we obtain
+the equivalence `equivalence : Karoubi (SimplicialObject C) ≌ Karoubi (ChainComplex C ℕ)`.
+It is in the namespace `CategoryTheory.Preadditive.DoldKan`. The functors in this
+equivalence are named `N` and `Γ`: by definition, they are `N₂` and `Γ₂`.
+
+In `EquivalencePseudoabelian.lean`, assuming `C` is idempotent complete,
+we obtain `equivalence : SimplicialObject C ≌ ChainComplex C ℕ`
+in the namespace `CategoryTheory.Idempotents.DoldKan`. This could be roughly
+obtained by composing the previous equivalence with the equivalences
+`SimplicialObject C ≌ Karoubi (SimplicialObject C)` and
+`Karoubi (ChainComplex C ℕ) ≌ ChainComplex C ℕ`. Instead, we polish this construction
+in `Compatibility.lean` by ensuring good definitional properties of the equivalence (e.g.
+the inverse functor is definitionallly equal to
+`Γ₀' : ChainComplex C ℕ ⥤ SimplicialObject C`) and
+showing compatibilities for the unit and counit isomorphisms.
+
+In this file `Equivalence.lean`, assuming the category `A` is abelian, we obtain
+`equivalence : SimplicialObject A ≌ ChainComplex A ℕ` in the namespace
+`CategoryTheory.Abelian.DoldKan`. This is obtained by replacing the functor
+`CategoryTheory.Idempotents.DoldKan.N` of the equivalence in the pseudoabelian case
+with the isomorphic functor `normalizedMooreComplex A` thanks to the isomorphism
+obtained in `Normalized.lean`.
+
+TODO: Show functoriality properties of the three equivalences above. More precisely,
+for example in the case of abelian categories `A` and `B`, if `F : A ⥤ B` is an
+additive functor, we can show that the functors `N` for `A` and `B` are compatible
+with the functors `SimplicialObject A ⥤ SimplicialObject B` and
+`ChainComplex A ℕ ⥤ ChainComplex B ℕ` induced by `F`. (Note that this does not
+require that `F` is an exact functor!)
+
+TODO: Introduce the degenerate subcomplex `D[X]` which is generated by
+degenerate simplices, show that the projector `PInfty` corresponds to
+a decomposition `K[X] ≅ N[X] ⊞ D[X]`.
+
+TODO: dualise all of this as `CosimplicialObject A ⥤ CochainComplex A ℕ`. (It is unclear
+what is the best way to do this. The exact design may be decided when it is needed.)
+
+## References
+* [Albrecht Dold, Homology of Symmetric Products and Other Functors of Complexes][dold1958]
+* [Paul G. Goerss, John F. Jardine, Simplicial Homotopy Theory][goerss-jardine-2009]
+
+-/
+
+
+noncomputable section
+
+open CategoryTheory Category Idempotents
+
+variable {A : Type _} [Category A] [Abelian A]
```

Comment:
With a view towards #6499, I suggest changing this to
```suggestion
variable {A : Type*} [Category A] [Abelian A]
```
Personally, I prefer being completely explicit when it comes to category theory universes (this will also look nicer once #6550 lands), but I understand that not everyone shares this opinion.